docs/extras/guide/generated_v1.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Generated API Python
+---
+
+The Python OCR SDK supports generated APIs.
+Generated APIs can theoretically support all APIs in a catch-all generic format.
+
+# Quick-Start
+
+```python
+from mindee import Client, product
+
+# Init a new client
+mindee_client = Client(api_key="my-api-key")
+
+# Add your custom endpoint (document)
+my_endpoint = mindee_client.create_endpoint(
+    account_name="my-account",
+    endpoint_name="my-endpoint",
+    version="my-version" # Note: version should be always provided when using for OTS products
+)
+
+# Load a file from disk
+input_doc = mindee_client.source_from_path("/path/to/the/file.ext")
+
+# Parse the file.
+# The endpoint must be specified since it cannot be determined from the class.
+result = mindee_client.parse(
+    product.GeneratedV1,
+    input_doc,
+    endpoint=my_endpoint
+)
+
+# Print a brief summary of the parsed data
+print(result.document)
+
+# Iterate over all the fields in the document
+for field_name, field_values in result.document.fields.items():
+    print(field_name, "=", field_values)
+```
+
+# Generated Endpoints
+
+You may have noticed in the previous step that in order to access a custom build, you will need to provide an account and an endpoint name at the very least.
+
+Although it is optional, the version number should match the latest version of your build in most use-cases.
+If it is not set, it will default to "1".
+
+# Field Types
+
+## Generated Fields
+
+### Generated List Field
+
+A `GeneratedListField` is a special type of custom list that implements the following:
+
+- **values** (`List[Union[StringField`[GeneratedObjectField](#Generated-object-field)`]]`): the confidence score of the field prediction.
```

Comment:
This is so that:
1) It will support it if it is implemented in the future
2) It will be properly added when using a regular OTS API with this class.